### PR TITLE
Add devcontainer config

### PR DESCRIPTION
**Pull request by mateuszlewko**

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "pytorch_xla",
+  "image": "us-central1-docker.pkg.dev/tpu-pytorch/docker/development:tpu",
+  "runArgs": [
+    "--privileged"
+  ],
+  "forwardPorts": [
+    4200
```
Comment (will-cromar) [inline .devcontainer/devcontainer.json:8]: What is this forwarded port for?

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  ],
+  "initializeCommand": "docker pull us-central1-docker.pkg.dev/tpu-pytorch/docker/development:tpu",
+  "mounts": [
+    "source=${localEnv:HOME}/.ssh,target=/home/vscode/.ssh,type=bind"
```
Comment (will-cromar) [inline .devcontainer/devcontainer.json:12]: This shouldn't be necessary. VSCode will forward your local SSH agent's keys automatically: https://code.visualstudio.com/remote/advancedcontainers/sharing-git-credentials#_using-ssh-keys

File: .devcontainer/devcontainer.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "pytorch_xla",
+  "image": "us-central1-docker.pkg.dev/tpu-pytorch/docker/development:tpu",
+  "runArgs": [
+    "--privileged"
+  ],
+  "forwardPorts": [
+    4200
+  ],
+  "initializeCommand": "docker pull us-central1-docker.pkg.dev/tpu-pytorch/docker/development:tpu",
```
Comment (will-cromar) [inline .devcontainer/devcontainer.json:10]: Does this update your local copy of the container to the latest one?

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (mateuszlewko) [reply .devcontainer/devcontainer.json]: Removed, thanks!

File: .devcontainer/devcontainer.json (hunk 3 above)
Comment (mateuszlewko) [reply .devcontainer/devcontainer.json:10]: Yes, this command runs before everything else.

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (mateuszlewko) [reply .devcontainer/devcontainer.json:8]: Forwarding the question to @stgpetrovic.

Comment (mateuszlewko) [reply .devcontainer/devcontainer.json]: Goran said it's sth for angular. I don't think it's needed, let's remove.

**Review by will-cromar — APPROVED** [review]

**Comment by will-cromar** [comment]
Thanks!

**Pull request merged**